Pegasus/include/pegasus/Collision.hpp

```diff
@@ -324,21 +320,21 @@ class Resolver
      * @param contacts contacts information
      * @param duration delta time of the frame
      */
-    void Resolve(std::vector<Contact>& contacts, double duration);
+    void Resolve(std::vector<Contact>& contacts, float duration);
 
     /**
      * @brief Resolves cached contacts
      *
      * @note This method is inteded to be called once during the pipeline execution
      */
```

Comment:
```suggestion
     *
     * @param duration delta time of the frame
     */
```

Reply:
Don't get it

Reply:
Doxygen is missing `@param duration` description for `ResolvePersistantContacts` method

Reply:
Tab indent